packages/story-editor/src/app/media/utils/useProcessMedia.js

```diff
@@ -248,15 +293,15 @@ function useProcessMedia({
             ...oldResourceWithoutId,
             trimData,
           },
-          originalResourceId: canvasResourceId,
+          originalResourceId: elementId,
```

Comment:
This now seems wrong. You are using a element id in place of a resource id.

Reply:
This one was going to ask you about and I want to dig a little further to understand that that is doing.

We have a reference to the "original_id: resourceId," under " additionalData" which is passed to that function.   So is "originalResourceId" just a temporary reference ?

```
 additionalData: {
            original_id: resourceId,
            web_stories_is_muted: isMuted,
            web_stories_media_source: isOptimized
              ? 'video-optimization'
              : 'editor',
          },
```

Reply:
Why was this changed? Why does `originalResourceId` have to be the elementId?